### PR TITLE
Cleanup references to past projects

### PR DESCRIPTION
**Pull request by kszaq**
If 11k lines dropped were not enough in https://github.com/LibreELEC/LibreELEC.tv/pull/3458, here are additional 46.

### CODE REVIEW

File: packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk [hunk 1]
```diff
@@ -17,4 +17,4 @@ PKG_LONGDESC="audiodecoder.usf"
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="kodi.audiodecoder"
-PKG_ADDON_PROJECTS="Generic Nvidia_Legacy RPi2 imx6 WeTek_Play"
+PKG_ADDON_PROJECTS="any"
```
Comment (jernejsk) [inline packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk]: You should just drop this line. No other addon defines it.

Comment (kszaq) [reply packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk]: Thanks, applied.

File: packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk [hunk 2]
```diff
@@ -17,4 +17,3 @@ PKG_LONGDESC="audiodecoder.usf"
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="kodi.audiodecoder"
-PKG_ADDON_PROJECTS="Generic Nvidia_Legacy RPi2 imx6 WeTek_Play"
```
Comment (MilhouseVH) [inline packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk]: Looks like the above excluded RPi1 for some reason (maybe doesn't build for armv6, or not enough CPU) but now we will build this add-on for RPi1, or at least try...

File: packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk (hunk 1 above)
Comment (MilhouseVH) [reply packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk]: > You should just drop this line. No other addon defines it.

```
neil@nm-linux:~/projects/pullrequest_repos/LibreELEC.tv$ git grep PKG_ADDON_PROJECTS=
config/docker/package.mk:PKG_ADDON_PROJECTS="@PROJECTS@"
packages/addons/libraries/adafruit-libraries/package.mk:PKG_ADDON_PROJECTS="RPi"
packages/addons/service/dispmanx_vnc/package.mk:PKG_ADDON_PROJECTS="RPi"
packages/addons/service/slice/package.mk:PKG_ADDON_PROJECTS="Slice Slice3"
packages/addons/service/touchscreen/package.mk:PKG_ADDON_PROJECTS="Generic RPi"
packages/addons/tools/rpi-tools/package.mk:PKG_ADDON_PROJECTS="RPi"
packages/packages.mk.addon_template:# PKG_ADDON_PROJECTS="[project, only set when not any]"
```

If we really wanted to exclude RPi (the device and not the project) then setting `PKG_ADDON_PROJECTS="!RPi1"` would suffice.

File: packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk (hunk 2 above)
Comment (MilhouseVH) [reply packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk]: Fixed in #3531 - this add-on builds with SSE2 or NEON with no way to disable NEON so it fails to build on armv6.

**Pull request merged**